### ISSUE TITLE
Remove user from Channel

### ISSUE THREAD

**Issue by rwakida** [sscpac/chat-locker#53]
As room owner, I can't find a way to remove a channel member.  From what I can tell, only the user can remove themselves from the room. See RocketChat/Rocket.Chat#168

**Comment by rwakida** [comment]
**update:** Removed private group from bug because we can remove users from private group using the relabeling changes.

**Comment by rwakida** [comment]
After consideration, this is probably not a big deal given our end users/environment.  @bbrockman should I close this?

**Comment by bbrockman** [comment]
nope, but you can keep it on the backlog.  Just don't put it on a Sprint and its fine.
